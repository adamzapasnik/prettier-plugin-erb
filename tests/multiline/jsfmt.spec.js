run_spec(__dirname, ['erb']);
