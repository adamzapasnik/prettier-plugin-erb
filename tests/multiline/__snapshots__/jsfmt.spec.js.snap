// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comments.html.erb 1`] = `
====================================options=====================================
parsers: ["erb"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<%#
This fails
  hey
    hey
      hey
  hey
%>

<%#
    This fails
    hey
      hey
        hey
    hey
%>

<%#
  This fails
  hey
    hey
      hey
  hey
%>

<%# This fails
  This fails
  hey
    hey
      hey
  hey
%>

<%# This fails
This fails
  hey
    hey
      hey
  hey
%>

<%# This fails
    This fails
    hey
      hey
        hey
    hey
%>

<%#This fails
  This fails
  hey
    hey
      hey
  hey
%>

<%#    This fails
  This fails
  hey
    hey
      hey
  hey
%>

<%#
    hey
%>

<%#
  hey
%>

<%#
hey
%>
=====================================output=====================================
<%#
  This fails
    hey
      hey
        hey
    hey
%>

<%#
  This fails
  hey
    hey
      hey
  hey
%>

<%#
  This fails
  hey
    hey
      hey
  hey
%>

<%#
  This fails
  This fails
  hey
    hey
      hey
  hey
%>

<%#
  This fails
  This fails
    hey
      hey
        hey
    hey
%>

<%#
  This fails
  This fails
  hey
    hey
      hey
  hey
%>

<%#
  This fails
  This fails
  hey
    hey
      hey
  hey
%>

<%#
  This fails
  This fails
  hey
    hey
      hey
  hey
%>

<%# hey %>

<%# hey %>

<%# hey %>

================================================================================
`;

exports[`formatted.html.erb 1`] = `
====================================options=====================================
parsers: ["erb"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div>
<%= react_component("HelloWorld", { greeting: "Hello from react-rails." }) %>

<%=
  react_component(
    "HelloWorld",
    { greeting: "Hello from react-rails." }
  ) %>
<%= react_component(
      "HelloWorld",
       { greeting: "Hello from react-rails." }
       ) %>

    <%= react_component(
      "HelloWorld",
       { greeting: "Hello from react-rails." }
       )
           %>
  <%=
 react_component(
      "HelloWorld",
    { greeting: "Hello from react-rails." },
    { greeting: "Hello from react-rails." },
    { greeting: "Hello from react-rails." },
    { greeting: "Hello from react-rails." }

       )
          %>

</div>

=====================================output=====================================
<div>
  <%= react_component('HelloWorld', { greeting: 'Hello from react-rails.' }) %>

  <%= react_component('HelloWorld', { greeting: 'Hello from react-rails.' }) %>
  <%= react_component('HelloWorld', { greeting: 'Hello from react-rails.' }) %>

  <%= react_component('HelloWorld', { greeting: 'Hello from react-rails.' }) %>
  <%=
    react_component(
      'HelloWorld',
      { greeting: 'Hello from react-rails.' },
      { greeting: 'Hello from react-rails.' },
      { greeting: 'Hello from react-rails.' },
      { greeting: 'Hello from react-rails.' },
    )
  %>
</div>

================================================================================
`;

exports[`long_deep_nested.html.erb 1`] = `
====================================options=====================================
parsers: ["erb"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<div>
<%=
react_component(
     "HelloWorld",
   { greeting: "Hello from react-rails react-rails react-rails react-rails react-rails react-rails react-rails." },
   { greeting: "Hello from react-rails react-rails react-rails react-rails react-rails react-rails react-rails." },
   { greeting: "Hello from react-rails." },
   { greeting: "Hello from react-rails." }

      )
         %>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>

=====================================output=====================================
<div>
  <div>
    <div>
      <div>
        <div>
          <div>
            <div>
              <div>
                <div>
                  <div>
                    <div>
                      <div>
                        <div>
                          <div>
                            <div>
                              <div>
                                <div>
                                  <div>
                                    <%=
                                      react_component(
                                        'HelloWorld',
                                        {
                                          greeting:
                                            'Hello from react-rails react-rails react-rails react-rails react-rails react-rails react-rails.',
                                        },
                                        {
                                          greeting:
                                            'Hello from react-rails react-rails react-rails react-rails react-rails react-rails react-rails.',
                                        },
                                        { greeting: 'Hello from react-rails.' },
                                        { greeting: 'Hello from react-rails.' },
                                      )
                                    %>
                                  </div>
                                </div>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>

================================================================================
`;

exports[`single.html.erb 1`] = `
====================================options=====================================
parsers: ["erb"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================





      <%= link_to "New Order",
       class: "btn btn-success" %>



=====================================output=====================================
<%= link_to 'New Order', class: 'btn btn-success' %>

================================================================================
`;
